server/container/guice/cassandra-guice/src/test/java/org/apache/james/JamesWithNonCompatibleElasticSearchServerTest.java

```diff
@@ -47,10 +48,11 @@
 
     @AfterAll
     static void afterAll() {
-        dockerES2.stop();
+        dockerES6.stop();
     }
 
     @Test
+    @Disabled("JAMES-3492, fails because communication fails with lower than 7.x nodes, maybe should be removed ")
```

Comment:
Or we should adapt the test to show which failure is precisely happening?

Reply:
This stopped failing, probaly IDE was using wrong classes, will remove Disabled